invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -21,6 +21,23 @@
     }
   }
 
+  /*Field Label*/
+  .ui.form .field .form-group label{
+    font-weight: normal;
+  }
+  // top level field
+  label.top-level-fields{
+    font-size: 14px;
+  }
+  // tob level field explicitly
+  label.top-level-fields[for="metadata.languages"]{
+    font-size: 14px;
+  }
+  // label without top-level field class
+  .required.field label[for="metadata.resource_type"]{
```

Comment:
Also here maybe we can apply a specific class like the above `top-level-fields`?